### PR TITLE
Feature login page

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Login Page added with 
- forgot password
- remember password
- create new account
temp-login-nou-store

### CODE REVIEW

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*Color and shades not finalised yet, will use color variables later*/
+.login-wrapper {
+  background-color: aliceblue;
+  margin: auto;
+  margin-top: 25vh;
```
Comment (JantuDeb) [inline css/login.css:5]: Instead of adding a `margin: 25vh` use `flex` to center it on the screen.

**Review by JantuDeb — APPROVED** [review]
Overall the code looks good I also added a suggestion you can look into

File: files/login.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <i class="fal fa-shopping-cart"> Cart</i>
+    </div>
+  </nav>
+
+  <div class="login-wrapper">
+    <h3>Login</h3>
+
+    <div class="input-div">
+      <label for="">Email</label>
+      <input type="email" />
```
Comment (Kothalkarpriya) [inline files/login.html:34]: Don't use empty attributes, if there is nothing you can put as of now remove it

File: files/login.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <label for="">Password</label>
+      <input type="password" />
+    </div>
+
+    <div class="checkbox-div">
+      <label for="">
```
Comment (Kothalkarpriya) [inline files/login.html]: everytime you used empty for remove it

**Review by Kothalkarpriya — APPROVED** [review]
Look into my suggestions after that you can merge

File: css/login.css (hunk 1 above)
Comment (abhijitdotsharma) [reply css/login.css:5]: I don't want the page to be centered for now, as I will be adding some footer and make the login page slightly left. Will make these changes then. Thanks for suggesting this.

**Pull request merged**